tgui/packages/tgui/interfaces/StaffWho.jsx

```diff
@@ -4,31 +4,24 @@ import { Window } from '../layouts';
 
 export const StaffWho = (props, context) => {
   const { data } = useBackend(context);
```

Comment:
useBackend no longer takes an argument.
```suggestion
  const { data } = useBackend();
```